### PR TITLE
adding maven support for vscode

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM registry.centos.org/centos/centos:7
 
 ENV LANG=en_US.UTF-8 \
     F8A_WORKER_VERSION=d403113 \
-    F8A_AUTH_VERSION=5211e23
+    F8A_AUTH_VERSION=5211e23 \
+    F8A_UTILS=33df0cc
```
Comment (miteshvp) [inline Dockerfile]: Is this intentional? See different versions of utils for RHEL and non-RHEL

Comment (yzainee) [reply Dockerfile]: no. it was a typo. rectified.

Comment (yzainee) [reply Dockerfile]: i didnt want to use the latest version and hence changed in dockerfile. forgot to change in rhel file

**Comment by miteshvp** [comment]
LGTM. @yzainee - feel free to merge

**Pull request merged**